README.md

```diff
@@ -89,6 +89,16 @@ mainnet merkle tree.
 
 ## Usage
 
+The airdrop tree was most recently updated on Dec 11, 2019.
```

Comment:
@pinheadmz we'll need to update the date before we merge this.

Reply:
I think I got that date from commits in this repo but it could be wrong.

Reply:
That date was correct when you opened this PR. The tree was updated at 9ab50eeb5b3f6ceb785cc94394facc6ea630096c.

Reply:
updated